.gitignore

```diff
@@ -1,4 +1,5 @@
 # Miscellaneous
+.gitignore
```

Comment:
Why should it be ignored?

Reply:
Sorry for that, it was just one test that I make and forgot to take out. In the new commit, I take that out.